client/lib/plans/features-list.tsx

```diff
@@ -610,7 +610,7 @@ export const FEATURES_LIST: FeatureList = {
 		getSlug: () => FEATURE_CUSTOM_DOMAIN,
 		getTitle: ( domainName?: string ) => {
 			if ( domainName ) {
-				return i18n.translate( 'The domain %(domainName)s is free for the first year', {
+				return i18n.translate( '%(domainName)s is included', {
```

Comment:
Hi @aneeshd16 just a random thought regarding this regarding the UX here. This feature will show the same `is included` even when a domain is mapped. Shouldn't this be `mapping` is included in the instance a domain is being mapped?

![image](https://github.com/Automattic/wp-calypso/assets/3422709/aea91fbc-9f13-4a08-9d5a-f6c1e154274f)